src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -425,4 +432,113 @@ public synchronized File getNextMixinMappings() {
 	public Set<File> getAllMixinMappings() {
 		return Collections.unmodifiableSet(mixinMappings);
 	}
+
+	public void runs(Closure<?> conf) {
+		runs.configure(conf);
+	}
+
+	public NamedDomainObjectContainer<RunConfigSettings> getRuns() {
+		return runs;
+	}
+
+	public static class RunConfigSettings implements Named {
```

Comment:
Would it be possible to expose a way to select a specific module (sourceSet) to use in this. Main use case is a `Test Minecraft Client/Server` run config for fabric api.

Reply:
Not sure how I would do that. There isn't really anything configurable in the generated run configs when it comes to source sets... Gotta experiment with this...

Reply:
Ok  I have a solution for this but I need to get the `Project` a `SourceSet` belongs to and that is apparently not easily done